app/assets/stylesheets/application.scss

```diff
@@ -14,6 +14,7 @@ $govuk-page-width: 1140px;
 @import "./admin/views/whats-new";
 @import "./admin/views/edit-edition";
 @import "./admin/views/attachments";
+@import "./admin/views/audit-trail";
```

Comment:
Just at a quick glance I notice we have a view CSS file here, but the CSS looks like it's global namespaced. 

I think we'll have to ponder whether this is a view's CSS or if it's a global module that can be used in the app.

This seems related to @dnkrj's comment.

Reply:
@dnkrj @kevindew I have scoped CSS with .audit-trail__history-comparison class